### PR TITLE
Added Release Notes for 4.7 ZStream

### PR DESCRIPTION
**Pull request by HarshCasper**
The PR has been reviewed by the SMEs and the QEs.

JIRA ID: [RHDEVDOCS-2789](https://issues.redhat.com/browse/RHDEVDOCS-2789) 

This PR is for OCP 4.7 ZStream. 

Direct Preview Link: https://deploy-preview-31166--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html

### CODE REVIEW

**Comment by christianvogt** [comment]
lgtm

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -1254,6 +1254,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
...
 
+* Previously, the API server could fail to create a resource and return a 409 status code due to a conflict while updating a resource quota resource. This issue has now been fixed and the OpenShift Console web app will attempt to retry the request while receiving a 409 status code. At most three attempts will be made which is often sufficient to complete the request. In the event that a 409 continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928228[*BZ#1928228*])
+
+* Previously, while creating a sample application the Developer Console created multiple resources. These resources depend on each other and should be created in a specific order. Due to this, an admission plugin fails to check one of these resources and the sample application could not be created. This issue has been fixed now, and instead of triggering all resources at the same time the code creates now all resources step by step, thus making a sample application more stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1933665[*BZ#1933665*])
```
Comment (rolfedh) [inline release_notes/ocp-4-7-release-notes.adoc:1259]: * While producing a sample app, the Developer Console creates multiple resources that depend on each other and must be completed in a specific order. Previously, an admission plugin sometimes could not check one of these resources, preventing the Developer Console from generating the Sample app. This issue has been fixed. The code creates the resources in the required order, so producing a sample app is more stable.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1933665[*BZ#1933665*])

Comment (rolfedh) [inline release_notes/ocp-4-7-release-notes.adoc:1259]:
```suggestion
* While producing a sample app, the Developer Console creates multiple resources that depend on each other and must be completed in a specific order. Previously, an admission plugin sometimes could not check one of these resources, preventing the Developer Console from generating the Sample app. This issue has been fixed. The code creates the resources in the required order, so producing a sample app is more stable.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1933665[*BZ#1933665*])
```

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -1254,6 +1254,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])
 
+* Previously, the API server could fail to create a resource and return a 409 status code due to a conflict while updating a resource quota resource. This issue has now been fixed and the OpenShift Console web app will attempt to retry the request while receiving a 409 status code. At most three attempts will be made which is often sufficient to complete the request. In the event that a 409 continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928228[*BZ#1928228*])
```
Comment (rolfedh) [inline release_notes/ocp-4-7-release-notes.adoc:1257]:
```suggestion
* Previously, the API server sometimes failed to create a resource and returned a 409 status code due to a conflict while updating a resource quota resource. This issue has been fixed. Now, if it receives a 409 status code, the OpenShift Console web app retries the request up to three times. If a 409 continues to occur, the console displays an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928228[*BZ#1928228*])
```

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (HarshCasper) [reply release_notes/ocp-4-7-release-notes.adoc:1259]: Thanks! Resolved.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (HarshCasper) [reply release_notes/ocp-4-7-release-notes.adoc:1257]: Thanks! Resolved.

**Comment by HarshCasper** [comment]
The PR is too messed up right now due to merge conflicts. I will make a fresh one.

**Pull request closed without merging**